### PR TITLE
Create README.md

### LINKED ISSUE #2: Meta: write a README
It would be good to add a README similar to other WHATWG standards.

### PR DESCRIPTION
**Pull request by saschanaz**
Fixes #2

Didn't include Tests because there's no dedicated test for this.

### CODE REVIEW

**Comment by jgraham** [comment]
Thanks for the PR!

I updated the README a bit to match TestUtils more closely (and remove the reference to IRC). Let me know if you think the changes are reasonable.

**Comment by saschanaz** [comment]
Test Utils or TestUtils? The title is currently Test Utils, so maybe do it everywhere?

**Comment by jgraham** [comment]
Good catch!

File: README.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ respect the [WHATWG Code of Conduct](https://whatwg.org/code-of-conduct).
...
 
-Folks notice minor and larger issues with the Test Utils Standard all the time and we'd love
-your help fixing those. Pull requests for typographical and grammar errors are also most welcome.
+We'd love your help fixing issues in the TestUtils standard. Pull requests for typographical and
+grammar errors are also most welcome.
 
 We'd be happy to mentor you through this process. If you're interested and need help getting
-started, leave a comment on the issue or ask around [on IRC](https://whatwg.org/irc).
+started, leave a comment on the issue or ask around 
+[on Matrix](https://matrix.to/#/#whatwg:matrix.org).
```
Comment (saschanaz) [inline README.md:15]: Heh, it seems DOM and Fullscreen should be fixed too!

**Comment by saschanaz** [comment]
LGTM with your change 👍

**Review by jgraham — APPROVED** [review]

**Pull request merged**